integration-tests-2.7/src/test/java/org/openmrs/module/fhir2/provider/r3/EncounterFhirResourceProvider_2_7IntegrationTest.java

```diff
@@ -37,7 +36,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.mock.web.MockHttpServletResponse;
 
-public class EncounterFhirResourceProvider_2_2IntegrationTest extends BaseFhirR3IntegrationTest<EncounterFhirResourceProvider, Encounter> {
+public class EncounterFhirResourceProvider_2_7IntegrationTest extends BaseFhirR3IntegrationTest<EncounterFhirResourceProvider, Encounter> {
```

Comment:
I had to move this to a new 2.7 package because tihs test will require the new "DECLINED" order fulfiller status